### PR TITLE
Create balance widget

### LINKED ISSUE #90: Create balance widget
A widget which can be seen throughout multiple screens and gets updated if the balance changes.

### PR DESCRIPTION
**Pull request by holzeis**
Screenshot 2022-10-31 at 12 51 57

resolves #90

### CODE REVIEW

**Review by da-kami — APPROVED** [review]
Sweet! Great that you added a test as well!

**Comment by da-kami** [comment]
Thought about merging this, but then saw that I better have a look at #91 first :)

File: integration_test/dashboard.dart [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import 'package:ten_ten_one/models/balance.model.dart';
+
+Widget createDashboard(balanceModel) => ChangeNotifierProvider<BalanceModel>(
+    create: (context) => balanceModel, child: const MaterialApp(home: Dashboard()));
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  group('Dashboard widget tests', () {
+    testWidgets('test if balance is rendered', (tester) async {
```
Comment (klochowicz) [inline integration_test/dashboard.dart]: 👏 thanks for writing the test!

File: pubspec.lock [hunk 2]
```diff
@@ -7,14 +7,14 @@ packages:
       name: _fe_analyzer_shared
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "49.0.0"
+    version: "50.0.0"
```
Comment (klochowicz) [inline pubspec.lock:10]: nit: tangling deps updates with feature work is best avoided - it makes it harder to find the cause of problems if some of them cause regressions.

**Review by klochowicz — APPROVED** [review]

**Pull request merged**